app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments/details/PaymentDetailExpandableCard.kt

```diff
@@ -54,16 +58,24 @@ internal fun PaymentDetailExpandableCard(
 ) {
   val dateTimeFormatter = rememberHedvigMonthDateTimeFormatter()
   HedvigCard(
-    onClick = onClick,
+    onClick = null,
     modifier = modifier,
   ) {
     Column(
       modifier = Modifier
         .padding(horizontal = 16.dp, vertical = 12.dp)
-        .fillMaxWidth(),
+        .fillMaxWidth()
+        .clickable(
+          interactionSource = remember { MutableInteractionSource() },
+          indication = rememberRipple(
+            bounded = false,
+            radius = 1000.dp),
```

Comment:
Yeah no problem with it being big if it does what we want.
Perhaps since it will be looking quite odd out of context write a little comment about it.
```suggestion
            // This fixes the problem of the ripple not properly resizing as the card expands
            radius = 1000.dp
          ),
```